sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -18,64 +18,50 @@
 
 
 class DictMixin(object):
-    def __setitem__(self, key, item):
-        # type: (Any, Any) -> None
+    def __setitem__(self, key: Any, item: Any) -> None:
         self.__dict__[key] = item
 
-    def __getitem__(self, key):
-        # type: (Any) -> Any
+    def __getitem__(self, key: Any) -> Any:
         return self.__dict__[key]
 
-    def __repr__(self):
-        # type: () -> str
+    def __repr__(self) -> str:
         return str(self)
 
-    def __len__(self):
-        # type: () -> int
+    def __len__(self) -> int:
         return len(self.keys())
 
-    def __delitem__(self, key):
-        # type: (Any) -> None
+    def __delitem__(self, key: Any) -> None:
         self.__dict__[key] = None
 
-    def __eq__(self, other):
-        # type: (Any) -> bool
+    def __eq__(self, other: Any) -> bool:
         """Compare objects by comparing all attributes."""
         if isinstance(other, self.__class__):
             return self.__dict__ == other.__dict__
         return False
 
-    def __ne__(self, other):
-        # type: (Any) -> bool
+    def __ne__(self, other: Any) -> bool:
         """Compare objects by comparing all attributes."""
         return not self.__eq__(other)
 
-    def __str__(self):
-        # type: () -> str
+    def __str__(self) -> str:
         return str({k: v for k, v in self.__dict__.items() if not k.startswith("_")})
 
-    def has_key(self, k):
-        # type: (Any) -> bool
+    def has_key(self, k: Any) -> bool:
         return k in self.__dict__
 
-    def update(self, *args, **kwargs):
-        # type: (Any, Any) -> None
+    def update(self, *args: Any, **kwargs: Any) -> None:
         return self.__dict__.update(*args, **kwargs)
 
-    def keys(self):
-        # type: () -> list
+    def keys(self) -> list:
```

Comment:
I'm not sure why we typed this with the standard collections `list`. However, [since we still support <=Python 3.8](https://stackoverflow.com/a/39458225/15364179), we'll need to import List from typing at the top and change this to:
```suggestion
    def keys(self) -> List:
```